pkg/apis/core/v1/conversion.go

```diff
@@ -41,11 +41,14 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 				"spec.restartPolicy",
 				"spec.schedulerName",
 				"spec.serviceAccountName",
+				"spec.securityContext.hostNetwork",
```

Comment:
I don't understand - this is not a real field in v1.  It's just `spec.hostNetwork`.  The internal name doesn't matter.

Reply:
done